### PR TITLE
Mention C support in near future in README.md

### PR DESCRIPTION
**Pull request by campospinto**
current README mentions possibility of selecting C as pyccel backend, but not always possible, see issue #431

### CODE REVIEW

**Review by yguclu — APPROVED** [review]

File: .github/workflows/continuous-integration.yml [hunk 1]
```diff
@@ -58,6 +58,7 @@ jobs:
       - name: Set default MPI and HDF5 C compilers on Ubuntu
         if: matrix.os == 'ubuntu-latest'
         run: |
+          sudo apt-get update
```
[the remark this answers is not in the document]
Comment (yguclu) [reply .github/workflows/continuous-integration.yml:61]: No, but it fixes the CI which started crashing two hours ago. Do you want me to create a separate PR?

[the remark this answers is not in the document]
Comment (yguclu) [reply .github/workflows/continuous-integration.yml:61]: See #440 😉

[the remark this answers is not in the document]
Comment (campospinto) [reply .github/workflows/continuous-integration.yml:61]: Thanks! I was just curious :)

**Comment by yguclu** [comment]
Have you merged this PR before PR #440? 🤦‍♂️

**Comment by campospinto** [comment]
> Have you merged this PR before PR #440? 🤦‍♂️

I thought you did the change here so that we don't have to wait for PR #440 to me merged. Wasn't that so?

**Comment by campospinto** [comment]
ah, ok now I see that the second comment was actually in response to your first one. Ok I read too fast.

**Comment by yguclu** [comment]
PR #440 contains no changes if merged after this PR, so it becomes useless. I will force a change in the history of `devel` just for this time.

**Comment by yguclu** [comment]
The merge commit in `devel`  corresponding to this PR has been force-removed. This PR is now superseded by #441.

**Pull request merged**